src/typegraph/core/src/types/sdk/core.rs

```diff
@@ -1,6 +1,7 @@
 // Copyright Metatype OÜ, licensed under the Mozilla Public License Version 2.0.
 // SPDX-License-Identifier: MPL-2.0
 
+use ordered_float::NotNan;
```

Comment:
How can we get a nan?

Reply:
By passing NaN from JS, I guess...
This is just to enable hashing... NaN can have a lot of different underlying values...

Reply:
I am guessing you got into that scenario this is why but, I am surprised we get a NaN out of a serde deserialized typegraph as it is not part of JSON spec.

Reply:
I did not get NaN. It is required for `#[derive(Hash)]`...